packages/rust-bindings/wasm.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module "*.wasm" {
```

Comment:
I think it would be good to add an explanation to this:
```suggestion
/**
 * Represents the format expected from importing `.wasm` files the have been pre-processed
 * to base64 strings as part of a build job.
 */
declare module "*.wasm" {
```